### ISSUE TITLE
Intruder lockout error

### ISSUE THREAD

**Issue by GuyKh** [yonatanp/electrasmart#4]
Getting `Intruder lockout` response.

Perhaps it's a mechanism of Electra for flooding.
Will try to reset the credentials. --> **UPDATE: it helped**

```
2020-11-11 11:47:47 DEBUG (SyncWorker_11) [electrasmart.client] Posting request
id: 1800
url: https://app.ecpiot.co.il/mobile/mobilecommand
headers: {'user-agent': 'Electra Client'}
post json data:
{'cmd': 'VALIDATE_TOKEN',
 'data': {'imei': '123123123',
          'os': 'android',
          'osver': 'M4B30Z',
          'token': '123123123123'},
 'id': 1800,
 'pvdid': 1,
 'sid': None}

2020-11-11 11:47:46 DEBUG (SyncWorker_11) [electrasmart.client] Response received (id=1863):
{'data': {'res': 1, 'res_desc': 'Intruder lockout', 'sid': None},
 'desc': None,
 'id': 1863,
 'status': 0}
```

**Comment by nitaybz** [comment]
I got it as well.... I lowered the frequency of the status polling and it seems better now

**Comment by yonatanp** [comment]
Should have been fixed by PR #7
